### ISSUE TITLE
Create playlists feature

### ISSUE THREAD

**Issue by Y0ngg4n** [jmshrv/finamp#458]
Add the feature to create a new playlist. Currently this is not supported.

**Comment by Chaphasilor** [comment]
I could design this for the redesign, in case you want to implement it yourself 😁

**Comment by Y0ngg4n** [comment]
@Chaphasilor  yes please :)

**Comment by Chaphasilor** [comment]
@Y0ngg4n btw, it is possible to create a playlist when adding a song to a playlist. On the playlist selection screen, there's a "+" button on the bottom right. But we could also show that button on the playlist list as well I guess

**Comment by pixel24** [comment]
> @Y0ngg4n btw, it is possible to create a playlist when adding a song to a playlist. On the playlist selection screen, there's a "+" button on the bottom right. But we could also show that button on the playlist list as well I guess

That's correct :-) However, the playlist is automatically set to "Public," and unfortunately, it can't be deleted in Finamp :-(

**Comment by flloschy** [comment]
There is a hackathon planed soon and I plan on always allowing playlist deletion, no promises though :)
In the mean time if you are using the beta you can enable Server-side deletion of media in `Settings>Interactions>Allow deletion from server`, this will allow you to delete playlists.

**Comment by flloschy** [comment]
@stephen304 Playlists can always be deleted from now on (if jellyfin think you are allowed to) :D
Well not now now, but next beta release
